src/resources/Enums.ts

```diff
@@ -132,48 +132,38 @@ export enum SourceStatusTypeWithTransition {
 }
 
 export enum SourceType {
-    ADOBE_EXPERIENCE_MANAGER = 'ADOBE_EXPERIENCE_MANAGER',
+    ADOBE_EXPERIENCE_MANAGER = 'ADOBE_EXPERIENCE_MANAGER', // @deprecated
```

Comment:
![image](https://github.com/user-attachments/assets/5f8787f5-54b2-4482-848c-dd7e6a510308)


```suggestion
    /**
    * @deprecated
    */
    ADOBE_EXPERIENCE_MANAGER = 'ADOBE_EXPERIENCE_MANAGER',
```

![image](https://github.com/user-attachments/assets/7518691f-6f36-46ec-916c-7ad048054957)